client/src/containers/NavigationItem/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import data from '../../constants/navigation-items';
+
+import Wrapper from './Wrapper';
+import Label from './Label';
+import Icon from './Icon';
+
+const WIDTH_LIMIT = 1024;
+
+const NavigationItem = ({ item, users, sidenav }) => {
+  const element = (
+    <Wrapper to={data[item].link} sidenav={sidenav ? 1 : 0} limit={WIDTH_LIMIT}>
+      <Icon name={data[item].icon} />
+      <Label sidenav={sidenav ? 1 : 0} limit={WIDTH_LIMIT}>{data[item].text}</Label>
```

Comment:
I made this to avoid this warning in the console:
![image](https://user-images.githubusercontent.com/26466644/48405325-5f6a8e80-e732-11e8-9c14-2f0a88543994.png)

I am not sure why, but I cannot pass boolean attribute to the dom element... 😞